CMakeLists.txt

```diff
@@ -285,6 +285,22 @@ endforeach()
 
 install(SCRIPT ${PROJECT_SOURCE_DIR}/cmake/GenerateHelptags.cmake)
 
+# MIN_LOG_LEVEL for log.h
+if(DEFINED MIN_LOG_LEVEL)
+  if(NOT MIN_LOG_LEVEL MATCHES "^[0-3]$")
+    message(FATAL_ERROR "MIN_LOG_LEVEL must be a number DEBUG(0), INFO(1), WARNING(2), ERROR(3)")
```

Comment:
To match local.mk (and for easier reading IMO), spaces between the parens (e.g., `WARNING (2)`) might be good,and an `or` before `ERROR`.